opentelemetry-user-events-logs/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 - Added support for Populating Cloud RoleName, RoleInstance from Resource's
   "service.name" and "service.instance.id" attributes respectively.
+- Make exporter reentrant-safe by removing logs that could be bridged back
+  to itself.
+- Exporter now unregisters the Tracepoint on `shutdown()`.
```

Comment:
Not fully aware of the implementation details. Is it `Provider` or `Tracepoint`?

```suggestion
- Exporter now unregisters the `Provider` on `shutdown()`.
```